pyproject.toml

```diff
@@ -32,29 +32,29 @@ test = [
     "pytest>=6.0",
     "pytest-xdist>=2.5",
     "pytest-cov>=3.0",
-    "MDAnalysisTests>=2.7.0",
-    "docker-py",
```

Comment:
Put docker-py back in as it is being used in the tests (`import docker`).

All packages that you need for testing need to be declared here.